src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -562,6 +564,60 @@ public void testForceOutput() {
         Assert.assertTrue(actualVC2.get(1).isPolymorphicInSamples());
     }
 
+    /**
+     * This tests whether NON_REF alleles are properly removed, including multi-allelic sites
+     */
+    @Test
+    public void testForceOutputNonRef() {
+        final File input = getTestFile("ForceOutputNonRef.g.vcf");
+        final File output1 = createTempFile("output", ".vcf");
+
+        // build intervals
+        final File output2 = createTempFile("output", ".vcf");
+        final File intervalFile = createTempFile("testForceOutputNonRef", ".intervals");
+        try (PrintWriter writer = new PrintWriter(IOUtil.openFileForWriting(intervalFile))) {
```

Comment:
Instead of creating a file, just specify multiple interval literals
```
. . .
   .addArgument(GenotypeGVCFs.FORCE_OUTPUT_INTERVALS_NAME, "1:1048236-1048236")
   .addArgument(GenotypeGVCFs.FORCE_OUTPUT_INTERVALS_NAME, "1:1051053-1051053")
   .addArgument(GenotypeGVCFs.FORCE_OUTPUT_INTERVALS_NAME, "1:2364622-2364622")
   . . .
```

Reply:
ok